tests/release/e2e-test-default-with-deployment.go

```diff
@@ -0,0 +1,231 @@
+package release
+
+import (
+	"os"
+	"strings"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
+	"knative.dev/pkg/apis"
+
+	ecp "github.com/hacbs-contract/enterprise-contract-controller/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	klog "k8s.io/klog/v2"
+)
+
+var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-e2e-with-deployment", Label("release", "withDeployment", "HACBS"), func() {
+	defer GinkgoRecover()
+	// Initialize the tests controllers
+	framework, err := framework.NewFramework()
+	Expect(err).NotTo(HaveOccurred())
+
+	var defaultEcPolicy = ecp.EnterpriseContractPolicySpec{}
+	var devNamespace = uuid.New().String()
+	var managedNamespace = uuid.New().String()
+
+	var cm = &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: buildPipelineBundleDefaultName,
+		},
+		Data: map[string]string{
+			"default_build_bundle": buildPipelineBundleDefault,
+		},
+	}
+
+	BeforeAll(func() {
+		kubeController := tekton.KubeController{
+			Commonctrl: *framework.CommonController,
+			Tektonctrl: *framework.TektonController,
+		}
+
+		_, err := framework.CommonController.CreateTestNamespace(devNamespace)
+		Expect(err).NotTo(HaveOccurred(), "Error when creating devNamespace: %v", err)
+		klog.Info("Dev Namespace created: ", devNamespace)
```

Comment:
we have replaced logging with klog with [GinkgoWriter](https://github.com/redhat-appstudio/e2e-tests/pull/297/commits/5a25c43a89f35a5eb23507948d9847430871d1b2) in tests. please use that one instead

Reply:
replaced all